src/app/child-dev-project/children/model/dateOfBirth.ts

```diff
@@ -0,0 +1,11 @@
+import { calculateAge } from "../../../utils/utils";
+
+/**
+ * Subclass of Date which provides the getter `age`.
+ * This age is calculated based on the date that this object represents.
+ */
+export class DateOfBirth extends Date {
```

Comment:
Maybe better named "DateWithAge" or similar? I can see a use for this also for admissionDate or similar properties.